Doc/faq/programming.rst

```diff
@@ -1739,7 +1739,8 @@ Variable names with double leading underscores are "mangled" to provide a simple
 but effective way to define class private variables.  Any identifier of the form
 ``__spam`` (at least two leading underscores, at most one trailing underscore)
 is textually replaced with ``_classname__spam``, where ``classname`` is the
-current class name with any leading underscores stripped.
+current class name with any leading underscores stripped
+(see :ref:`here <private-name-mangling>` for details and special cases).
```

Comment:
Some comments:

1. In general it is good practice to make the link text meaningful on its own; this can be helpful for, for example, braille readers. IOW, _"here"_ won't pass 😉
2. Often, the `.. sealso::` directive is a good fit for cases like this.

Reply:
I wanted to avoid the `see also` because would break the reading flow IMO. On the other hand, the best title I could think of was `See the private identifier specifications` but I was not really sure whether it was the best or not.

Reply:
Actually, the `seealso` is fine visually so I'll use it more in the future:

![image](https://github.com/python/cpython/assets/10796600/92616a54-3e87-4fcb-9b71-5929ddddf5fa)

I am a bit biased with the default themes in Sphinx where the See Also block is a very huge block...